samcli/commands/build/command.py

```diff
@@ -335,3 +370,42 @@ def _get_mode_value_from_envvar(name: str, choices: List[str]) -> Optional[str]:
         raise click.UsageError("Invalid value for 'mode': invalid choice: {}. (choose from {})".format(mode, choices))
 
     return mode
+
+
+def _process_env_var(container_env_vars: Optional[List[str]]) -> Dict:
+    """
+    Parameters
+    ----------
+    container_env_vars : list
+        the list of command line env vars received from --container-env-vars flag
+        Each input format needs to be either function specific format (FuncName.VarName=Value)
+        or global format (VarName=Value)
+
+    Returns
+    -------
+    dictionary
+        Processed command line environment variables
+    """
+    processed_env_vars: Dict = {}
+
+    if container_env_vars:
+        for env_var in container_env_vars:
+            location_key = "Parameters"
+            pure_env_var = env_var
+            if "." in env_var:
+                test_target = env_var.split(".")
```

Comment:
What's a test target?

Reply:
a target string to be tested on its format